xml/src/main/java/org/ehcache/config/xml/XmlConfiguration.java

```diff
@@ -156,7 +157,21 @@ private void parseConfiguration()
           cacheClassLoader = ClassLoading.getDefaultClassLoader();
         }
       }
-      
+      if(cacheDefinition.persistenceMode() != null) {
+        CacheConfiguration.PersistenceMode mode = null;
+        switch (cacheDefinition.persistenceMode()) {
+        case CREATE_IF_ABSENT: 
+          mode = CacheConfiguration.PersistenceMode.CREATE_IF_ABSENT ;
+          break;
+        case EXPECT_EXISTS:
+          mode = CacheConfiguration.PersistenceMode.EXPECT_EXISTS;
+          break;
+        default:
```

Comment:
Call me paranoid, but I think an XML config file that doesn't specify the XSD is going to be accepted by the code. Plus if the XSD ever gets out of sync with the code sometime in the future (think of a bug in a future code change) then the bug would be a lot easier to track down.
Or do you think I'm being unreasonably paranoid?